perfzero/lib/perfzero/perfzero_config.py

```diff
@@ -215,6 +225,11 @@ def add_benchmark_parser_arguments(parser):
       default=None,
       type=str,
       help='A string that uniquely identifies the benchmark execution.')
+  parser.add_argument(
+      '--result_uploader_methods',
```

Comment:
I am wondering if `--result_upload_methods` is more consistent with the existing configs such as `--benchmark_methods`.

Reply:
Done